packages/ui/package.json

```diff
@@ -27,6 +27,7 @@
     "abortable-iterator": "^3.0.0",
     "bufferutil": "^4.0.3",
     "it-pipe": "^1.1.0",
+    "daemon": "0.0.0",
```

Comment:
This is kind of a bummer. Lerna only does the right thing if we have a x.x.x version here, but now running npm install [something] in the ui package will fail because it's looking for daemon v0.0.0 in the registry. :(

Reply:
:(

What if we create an NPM script in the daemon package that generates the JSON file containing its dependencies? We could then merge the two JSON files.

Failing that, I think it may be best to abandon the monorepo approach with lerna and just switch back to a single package.

Thoughts?

Reply:
I found a workaround in the latest commit, so let's see how that goes for now.

Reply:
I do find it nice to have separate packages/configs/etc...